unison-src/transcripts/api-getDefinition.output.md

```diff
@@ -217,4 +217,100 @@ GET /api/getDefinition?names=x&relativeTo=nested
+            "bestTermName": "x",
+            "defnTermTag": null,
+            "signature": [
+                {
+                    "annotation": {
+                        "contents": "##Nat",
+                        "tag": "TypeReference"
+                    },
+                    "segment": "Nat"
+                }
+            ],
+            "termDefinition": {
+                "contents": [
+                    {
+                        "annotation": {
+                            "contents": "x",
+                            "tag": "HashQualifier"
+                        },
+                        "segment": "x"
+                    },
+                    {
+                        "annotation": {
+                            "tag": "TypeAscriptionColon"
+                        },
+                        "segment": " :"
+                    },
+                    {
+                        "annotation": null,
+                        "segment": " "
+                    },
+                    {
+                        "annotation": {
+                            "contents": "##Nat",
+                            "tag": "TypeReference"
+                        },
+                        "segment": "Nat"
+                    },
+                    {
+                        "annotation": null,
+                        "segment": "\n"
+                    },
+                    {
+                        "annotation": {
+                            "contents": "x",
+                            "tag": "HashQualifier"
+                        },
+                        "segment": "x"
+                    },
+                    {
+                        "annotation": {
+                            "tag": "BindingEquals"
+                        },
+                        "segment": " ="
+                    },
+                    {
+                        "annotation": null,
+                        "segment": " "
+                    },
+                    {
+                        "annotation": {
+                            "tag": "NumericLiteral"
+                        },
+                        "segment": "42"
+                    }
+                ],
+                "tag": "UserObject"
+            },
+            "termDocs": [
+                [
+                    "doc",
+                    "#ulr9f75rpcrv79d7sfo2ep2tvbntu3e360lfomird2bdpj4bnea230e8o5j0b9our8vggocpa7eck3pus14fcfajlttat1bg71t6rbg",
+                    {
+                        "contents": [
+                            {
+                                "contents": "Documentation",
+                                "tag": "Word"
+                            }
+                        ],
+                        "tag": "Paragraph"
+                    }
+                ]
+            ],
+            "termNames": [
+                ".nested.names.x",
```

Comment:
After a conversation with Simon we determined we should only have a single name per definition in this list, the name should be relative to the `relativeTo` path when possible, and be an absolute name otherwise.